cypress/integration/details_spec.js

```diff
@@ -91,3 +95,18 @@ describe('Details view error handling - Money Plane', () => {
   })
 
 });
+
+describe('Details view error handling for 500 status code - Money Plane', () => {
+
+  const baseUrl = 'http://localhost:3000/694919'
+
+  it ('Shows an error for a 500 status code response', () => {
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/694919', {
+      statusCode: 500,
+    })
+
+    cy.visit(baseUrl)
+      .get('.error-message').should('contain', 'These taters got too rowdy - check back later!')
+  })
+
+});
```

Comment:
Cool, I don't know that we needed any different error message to display here, but like this additional check on the specific 500 status code.